app/views/ReportView/components/KbMatches/index.tsx

```diff
@@ -60,7 +60,7 @@ const KB_MATCHES_TITLE_MAP = {
 
 const RAPID_TABLE_TITLE_MAP = {
   // Should be therapeuticAssociation, but the tag in backend is looking for 'therapeutic'
-  therapeutic: 'Variants with Clinical Evidence for Treatment in This Tumour Type',
+  therapeuticAssociation: 'Variants with Clinical Evidence for Treatment in This Tumour Type',
```

Comment:
the reason why it's `therapeutic` instead of `therapeuticAssociation` was due to 
```
    "kbMatches": [
        {
            "category": "therapeutic",
      ...
```